### PR TITLE
docs: Various LRP gsg fixups

### PR DESCRIPTION
**Pull request by nebril**

### CODE REVIEW

File: Documentation/gettingstarted/local-redirect-policy.rst [hunk 1]
```diff
@@ -459,6 +463,16 @@ security credentials for pods.
   .. code-block:: bash
 
       $ kubectl exec app-pod -- curl -s -w "\n" -X GET http://169.254.169.254/latest/meta-data/
+      ami-id
```
Comment (joestringer) [inline Documentation/gettingstarted/local-redirect-policy.rst:451]: If we'll show the output (which I think is good), I think we should also fix the syntax highlighting for this block: `.. code-block:: shell-session`

**Review by joestringer — APPROVED** [review]
Changes LGTM, one syntax highlighting not below.

**Review by aditighag — APPROVED** [review]

File: Documentation/gettingstarted/local-redirect-policy.rst (hunk 1 above)
Comment (aditighag) [reply Documentation/gettingstarted/local-redirect-policy.rst:451]: Done

File: Documentation/gettingstarted/local-redirect-policy.rst [hunk 2]
```diff
@@ -412,21 +412,19 @@ security credentials for pods.
...
-    running requests to the metadata server, then you may need to whitelist the
-    metadata requests by passing the below argument to the ``kiam-agent Deamonset``.
+  - You may see an error like "request blocked by whitelist-route-regexp" while
+    running requests to the metadata server once the kiam-agent pods are deployed.
+    In such cases, you may need to whitelist the metadata requests.
 
     .. code-block:: bash
 
-        $ sed -i '/args:/a \ \ \ \ \ \ \ \ \ \ \ \ - --whitelist-route-regexp=meta-data' kiam.yaml
+        $ helm template --set agent.extraArgs.whitelist-route-regexp=meta-data kiam uswitch/kiam > kiam.yaml
```
Comment (joestringer) [inline Documentation/gettingstarted/local-redirect-policy.rst]: Reading the overall list of items here, we have 

- Deploy
  - Instructions add the repository, but don't actually deploy
- You may see an error....
   - Template instructions
- ...
- Apply the configuration
  - Finally, the step where we deploy.

Which seems a bit odd in terms of ordering if you just read through?

Also, now that we don't need to run the extra `sed` commands, we could just do `helm install` directly with this option?

I'm wondering whether it's even worth mentioning things like the "request blocked while whitelist-out-regexp" bullet point above or the "iptables" bullet point below if we just tell people to install with the following command:

```
   helm install --set agent.host.iptables=false --set agent.extraArgs.whitelist-route-regexp=meta-data kiam uswitch/kiam
```

We can always add items afterwards to explain why each setting is configured like this, that's fine but I feel like we should provide the exact set of options that we expect users to deploy with to have the best chance of success. Then if a user deviates from these instructions, they can consult the documentation and see exactly which options we recommended.

**Review by joestringer — CHANGES_REQUESTED** [review]
Nice find, this is much better without the ugly sed command. Minor nit below.

**Review by joestringer — APPROVED** [review]

**Pull request merged**